pkg/config/config.go

```diff
@@ -1215,6 +1215,7 @@ func InitConfig(config Config) {
 	config.BindEnvAndSetDefault("serverless.logs_enabled", true)
 	config.BindEnvAndSetDefault("enhanced_metrics", true)
 	config.BindEnvAndSetDefault("capture_lambda_payload", false)
+	config.BindEnvAndSetDefault("capture_payload_max_depth", 10)
```

Comment:
Could we make this `capture_lambda_payload_max_depth`? That way it more closely matches the other env var.

Reply:
Updated.